webapp/security/load_fixtures.py

```diff
@@ -0,0 +1,18 @@
+import yaml
+
+from webapp.database import db_session
+from webapp.models import Release
+
+
+with open("webapp/security/fixtures/releases.yaml") as file:
+    document = yaml.full_load(file)
+    for codename, r in document["releases"].items():
+        release = Release(
+            name=r["name"],
+            version=r["version"],
+            codename=codename,
+            lts=r["lts"],
+        )
+
+        db_session.add(release)
+        db_session.commit()
```

Comment:
so every time we start the application we are updating the releases db?

Reply:
No, every time you run that script, it was mostly a helper thing for me to bootstrap the DB

Reply:
I can wrap it in a function if we decide to keep it